src/Nethermind/Nethermind.Runner/Program.cs

```diff
@@ -210,23 +210,15 @@ private static void Run(string[] args)
                 catch (Exception e)
                 {
                     if (_logger.IsError) _logger.Error("Error during ethereum runner start", e);
-                    if (e is IExceptionWithExitCode withExit)
-                    {
-                        exitCode = withExit.ExitCode;
-                    }
-                    else
-                    {
-                        exitCode = ExitCodes.GeneralError;
-                    }
-                    _processCloseCancellationSource.Cancel();
+                    _processExitSource.ExitCode = e is IExceptionWithExitCode withExit ? withExit.ExitCode : ExitCodes.GeneralError;
```

Comment:
Now setting the `ExitCode` won't cances the source. Previously, there was `Cancel` call. Is this a problem? If no, can you elaborate why?

Reply:
Right got confused.